### PR TITLE
add chalk source code for sagemaker

### PR DESCRIPTION
**Pull request by amy-chalk**

### CODE REVIEW

File: full_examples/sagemaker/src/models.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
+from datetime import date
+from dateutil.relativedelta import relativedelta
+from chalk import online, DataFrame, FeatureTime, Windowed, _, feature, windowed
+from chalk.features import features
+
+
+@features
+class Transaction: id: int
```
Comment (sjmignot) [inline full_examples/sagemaker/src/models.py:8]:
```suggestion
class Transaction: 
    id: int
```

File: full_examples/sagemaker/src/resolvers/customers.chalk.sql [hunk 2]
```diff
@@ -0,0 +1,11 @@
+-- resolves: customer
```
Comment (sjmignot) [inline full_examples/sagemaker/src/resolvers/customers.chalk.sql:1]:
```suggestion
-- resolves: Customer
```

File: full_examples/sagemaker/src/resolvers/transactions.chalk.sql [hunk 3]
```diff
@@ -0,0 +1,10 @@
+-- resolves: transaction
```
Comment (sjmignot) [inline full_examples/sagemaker/src/resolvers/transactions.chalk.sql:1]:
```suggestion
-- resolves: Transaction
```

File: full_examples/sagemaker/src/models.py (hunk 1 above)
Comment (amy-chalk) [reply full_examples/sagemaker/src/models.py:8]: come on amy!!!

**Review by sjmignot — APPROVED** [review]

**Pull request merged**